tests/Unit/ItemTest.php

```diff
@@ -4312,6 +4312,93 @@ public static function getTitleDataProvider(): array
                 ,
                 'Item Title',
             ],
+            'Test RSS 0.91 XML and DTD' => [
+<<<XML
+<?xml version="1.0"?>
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>Item Title</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'Item Title',
+            ],
+            'Test RSS 0.91 XML and DTD and external HTML entity' => [
+<<<XML
+<?xml version="1.0"?>
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>caf&eacute;</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'café',
+            ],
+            'Test RSS 0.91 DTD and external HTML entity' => [
+<<<XML
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>caf&eacute;</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'café',
+            ],
+            'Test RSS 0.91 internal DTD with internal entity' => [
+<<<XML
+<!DOCTYPE rss [
+	<!ENTITY helloworld "Hello World!">
+]>
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>&helloworld;</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'Hello World!',
+            ],
+            'Test RSS 0.91 XML and undeclared HTML entity (invalid)' => [
+<<<XML
+<?xml version="1.0"?>
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>caf&eacute;</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'café',
+            ],
+            'Test RSS 0.91 with undeclared HTML entity (invalid)' => [
+<<<XML
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>caf&eacute;</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'café',
+            ],
```

Comment:
In theory, the following should work as well but it is probably to niche to care about:

```suggestion
            ],
            'Test RSS 0.91 DTD and external HTML entity and internal entity' => [
<<<XML
<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd" [
    <!ENTITY adios "Adi&oacute;s">
]>
<rss version="0.91">
    <channel>
        <item>
            <title>&adios; Adi&oacute;s</title>
        </item>
    </channel>
</rss>
XML
                ,
                'Adiós Adiós',
            ],
```

Reply:
Indeed, but merging DTDs do not look easy as they may contain more complicated syntax